index.js

```diff
@@ -57,51 +98,98 @@
 // Rock, Paper, Sissors
 // Your function should take a string (either rock paper or sissors)
 // it should return you won or you lost based on the rules of the game (you may need to look up the rules if you have not played before)
-// use math.random to determine the computers choice 
-// hint while you can complete this with only conditionals based on strings it may help to equate choice to a number 
-
-  
-  
+// use math.random to determine the computers choice
+// hint while you can complete this with only conditionals based on strings it may help to equate choice to a number
+function playGame(player) {
+
+  let computer = Math.random()
+  if (computer <= 0.30) {
+    computer = "rock"
+  } else if (computer <= 0.60) {
+    computer = "paper"
+  } else if (computer > 0.60) {
+    computer = "scissors"
+  }
+
+  let result = null;
+
+  if (player === "rock" && computer === "rock" || player === "paper" && computer === "paper" || player === "scissors" && computer === "scissors") {
+    result = "tie"
+  } else if (player === "rock" && computer === "scissors" || player === "paper" && computer === "rock" || player === "scissors" && computer === "paper") {
+    result = "win"
+  } else if (player === "scissors" && computer === "rock" || player === "rock" && computer === "paper" || player === "paper" && computer === "scissors") {
+    result = "lose"
+  }
+
+  return result;
+}
+var player = prompt("Choose");
+console.log(playGame(player))
 
 /************************************************************** Task 5 **************************************************************/
 //Metric Converter
 //a. KM to Miles - should take the number of kilometers and convert it to the equal number of miles
 
-
+var kilometers = parseInt(5);
+var miles = kilometers / 1.6;
+console.log(miles + " Miles");
 
 
 
 //b. Feet to CM - should take the number of feet and convert it to the equal number of centimeters
-  
 
 
+var feet = parseInt(5);
+var cm = feet * 30.48;
+console.log(cm + " CM");
 
 
 /************************************************************** Task 6 **************************************************************/
 // 99 bottles of soda on the wall
 // create a function called annoyingSong
 // the function should take a starting number as an argument and count down - at each iteration it should log (number) bottles of soda on the wall, (number) bottles of soda, take one down pass it around (number left over) bottles of soda on the wall`
-  
+
+function annoyingSong(number) {
+  let = number
+  for (let i = number; i >= 0; i--) {
+    console.log(i)
```

Comment:
Be sure to double-check the instructions. It's asking you to console.log `(number) bottles of soda on the wall, (number) bottles of soda, take one down pass it around (number leftover) bottles of soda on the wall`

Aside from that, your condition looks great inside of your for loop.